CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## Unreleased
 * Fixed an issue where the error was not clear when trying to retrieve the server version.
 * Fixed an issue in **prepare-content** where tags were added to metadata because of test scripts.
+* Updated demisto-sdk coverage-analyze to print a warning and exit with exit code 0 in case that the .coverage file doesn't exist.
```

Comment:
```suggestion
* Fixed an issue where the **coverage-analyze** would crash in case the `.coverage` file does not exist
```

wouldn't it make more sense?